src/step-3-logging/routes/users.js

```diff
@@ -0,0 +1,7 @@
+export default async function users(fastify) {
+  fastify.get('/users', async () => {
+    fastify.log.info('Users route called')
```

Comment:
```suggestion
  fastify.get('/users', async (req) => {
    req.log.info('Users route called')
```

This is a critical change. `req.log` is a child logger in pino that automatically includes the request id. In this way, a user can correlate all log lines for a given request. We should update this everywhere and cover it in the slides.